nomenclature/code.py

```diff
@@ -209,3 +210,17 @@ class RegionCode(Code):
     """
 
     hierarchy: str = None
+    countries: List[str] = None
+
+    @validator("countries")
+    def check_iso3_codes(cls, v, values) -> bool:
+        invalid_iso3_codes: List[str] = []
+        for country in v:
+            if pycountry.countries.get(alpha_3=country) is None:
+                invalid_iso3_codes.append(country)
+        if invalid_iso3_codes:
+            raise ValidationError(
```

Comment:
This needs to be a `ValueError` (or any kind of other error except `pydantic.ValidationError`:
```suggestion
            raise ValueError(
```
Reason being, pydantic collects all the errors found during validation and raises them as one common error which is a `pydantic.ValidationError`. The individual errors, however, cannot be `pydantic.ValidationError`.